log4j-slf4j2-impl/src/test/java/org/apache/logging/slf4j/LoggerContextTest.java

```diff
@@ -35,9 +35,9 @@ public void testCleanup() throws Exception {
         factory.getLogger("test");
         Set<LoggerContext> set = factory.getLoggerContexts();
         final LoggerContext ctx1 = set.toArray(LoggerContext.EMPTY_ARRAY)[0];
-        assertTrue("LoggerContext is not enabled for shutdown", ctx1 instanceof LifeCycle);
+        assertTrue(ctx1 instanceof LifeCycle, "LoggerContext is not enabled for shutdown");
```

Comment:
```suggestion
        assertInstanceOf(LifeCycle.class, ctx1, "LoggerContext is not enabled for shutdown");
```

Could you change the assertions on `X instanceof Y` to use the new `assertInstanceOf` assertions?